src/main/resources/db/changelog/changes/02-change-price-type-to-decimal.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: change-price-type-to-decimal
+      author: your_name
+      changes:
+        - modifyDataType:
+            tableName: books
+            columnName: price
+            newDataType: decimal(10, 2)
```

Comment:
first of all - you do not have this script in the database changelog, so it is not applied to the DB.
And I'd rather try to keep the liquibase scripts as one per PR to avoid having too many of them in the main branch over time